entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -65,6 +65,9 @@ in the current directory, but can be changed using the <nobr>**--project**</nobr
 The startup project is the one emulated by the tools when executing your project's code. It also defaults to the project
 in the current directory, but can be changed using the **--startup-project** option.
 
+> [!NOTE]
+> Running a `database update` for an ASP.NET Core project that has EF Core installed in a different class library would look like this: `dotnet ef --startup-project {Web-Project-Path} database update` (from the library directory that has the CLI tool installed)
+
```

Comment:
I always imagined invoking the command from the startup project would be a lot more common:

    MyProject.Web> dotnet ef database update --project ..\MyProject.Data

And in the case your context is registered in the application services, you wouldn't even need to specify `--project`.

Reply:
@bricelam That makes a lot of sense. I'll change the note to reflect your observation.
I think is good to be explicit about this because is such a common scenario, you know.

Reply:
@bricelam any more comments on this?

Reply:
@divega This is outdated, look for the updated version. @bricelam is there anything else to add?